source/Octopus.Server.Client/Model/AuthenticationConfigResource.cs

```diff
@@ -35,5 +35,8 @@ public AuthenticationConfigResource()
 
         [Writeable]
         public bool RememberMeEnabled { get; set; }
+
+        [Writeable]
+        public bool ApiKeysEnabled { get; set; }
```

Comment:
The year is 2023. Secops want to disable API keys, not just for service accounts. What is that new property called?

Reply:
👍 Changed to `ApiKeysEnabledForNonServiceAccounts`